pkgs/tools/admin/auth0-cli/default.nix

```
@@ -0,0 +1,36 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "auth0-cli";
+  version = "0.11.10";
+
+  src = fetchFromGitHub {
+    owner = "auth0";
+    repo = "auth0-cli";
+    rev = "v${version}";
+    hash = "sha256-1/T2hpSNamorWFuaSBoLsGpe9I06HGew9S3yJsDLmLQ=";
+  };
+
+  vendorHash = "sha256-d9ZwK/LAZGgeagGsg3bGYnVykfQcCLUex0pe/PUCtkA=";
+
+  ldflags = [
+    "-s" "-w"
+    "-X github.com/auth0/auth0-cli/internal/buildinfo.Version=v${version}"
+  ];
```

I would've also added [other](https://github.com/auth0/auth0-cli/blob/main/.goreleaser.yml#L15-L19) ldflags flags here as well. Although, quickly looking at the source code, only `Version` and `Revision` are actually used.

It can be done by setting `leaveDotGit` and extracting the actual commit revision, but to keep it simple, you can just put a mock one like so:
```suggestion
  ldflags = [
    "-s" "-w"
    "-X github.com/auth0/auth0-cli/internal/buildinfo.Version=v${version}"
    "-X github.com/auth0/auth0-cli/internal/buildinfo.Revision=0000000"
  ];
```